proto_files/text_classifier_pb2.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file shouldn't be added.

Reply:
Removed.... shouldn't this be in .gitignore?

Reply:
It is in .gitignore :smile: